magento/magento2
# #36947 Fixed timezone of order_date field returned from graphql customerOrders query

### Description (*)

The PR changes `order_date` field in `customerOrders` query to return date using store timezone instead of UTC.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#36947

### Manual testing scenarios (*)

1. Configure your store with a timezone different from +0
2. Create a Shipment with a comment
3. Verify the comment created_at in the normal view phtml in the website
4. Call customer Orders query and check created_at field from shipment comment

### Questions or comments

I believe no tests other than manual should be required here

### Contribution checklist (*)
